lms/static/scripts/frontend_apps/api-types.ts

```diff
@@ -312,6 +312,7 @@ export type StudentGradingSyncStatus = 'in_progress' | 'finished' | 'failed';
 export type StudentGradingSync = {
   h_userid: string;
   status: StudentGradingSyncStatus;
+  grade: number;
```

Comment:
As discussed [here](https://github.com/hypothesis/lms/pull/6749#discussion_r1789721220), adding one missing prop from the type definition which makes the naming more clear